WORKSPACE

```diff
@@ -52,3 +52,13 @@ http_archive(
     url = "https://github.com/bazelbuild/bazel/releases/download/0.9.0/bazel-0.9.0-dist.zip",
     sha256 = "efb28fed4ffcfaee653e0657f6500fc4cbac61e32104f4208da385676e76312a",
 )
+
+git_repository(
```

Comment:
add a comment explaining why we depend on this. Does it affect users? Do they need to add this to their WORKSPACE as well?